dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/Application/Application.cs

```diff
@@ -626,5 +628,38 @@ private Task _StartLongRunningCall(ITurnContext turnContext, Func<ITurnContext,
                 return handler(turnContext, cancellationToken);
             }
         }
+
+        internal static T? GetInvokeValue<T>(IInvokeActivity activity)
```

Comment:
Consider moving these helpers to a separated class.

Reply:
Moved to a Utilities class.